website/addons/osfstorage/model.py

```diff
@@ -423,17 +392,22 @@ class OsfStorageGuidFile(GuidFile):
 
     path = fields.StringField(required=True, index=True)
```

Comment:
@chrisseto Could you please document this field? The semantics of `path` seems to differ in subtle ways for different addons, so it'd be good to start documenting this. Even an example value goes a long way.